prow/plugins/respond.go

```diff
@@ -23,7 +23,7 @@ import (
 	"k8s.io/test-infra/prow/github"
 )
 
-const AboutThisBot = "Instructions for interacting with me using PR comments are available [here](https://github.com/kubernetes/community/blob/master/contributors/devel/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://github.com/kubernetes/test-infra/blob/master/commands.md)."
```

Comment:
@rmmh i think i got your meaning. for other comments, the link should still be there. am i right about this?
i thought we are removing the link for _all_ comments.